web/impact/impact/tests/test_create_edit_office_hours_view.py

```diff
@@ -2,13 +2,14 @@
 from datetime import datetime, timedelta
 from django.core import mail
 from django.urls import reverse
-
+import swapper
 from accelerator.models import MentorProgramOfficeHour, UserRole
 from accelerator.tests.contexts import UserRoleContext
 from accelerator.tests.factories import (
     MentorProgramOfficeHourFactory,
     ProgramFactory
 )
+from accelerator.apps import AcceleratorConfig
```

Comment:
I commented on this earlier -- since we're only pulling this in to reference `AcceleratorConfig.name`, and that is always (unless I'm missing something) set to `"accelerator"`, can we use the literal instead and drop the imports?

Reply:
(We've discussed this and agreed that factoring this out deserves its own ticket -- there are over 100 `AcceleratorConfig` references.)

Reply:
https://masschallenge.atlassian.net/browse/AC-7856